modules/api/src/main/java/org/apache/fluo/api/client/LoaderExecutor.java

```diff
@@ -48,11 +48,12 @@
 
   /**
    * Same as {@link #execute(Loader)} except it returns a future that completes upon successful
-   * commit and if an exception is thrown in the loader, it will be relayed through the future.
+   * commit and if an exception is thrown in the loader, it will be relayed through the future. The
+   * result of the future is the Loader that was successfully executed.
```

Comment:
This comment may need to be formatted/wrapped.  Trying to communicate that loaders may executor one or more times.

```suggestion
   * result of the future is the Loader that was successfully executed. If storing any information in the loader, keep in mind that loader may be automatically executed multiple times in the case of commit collisions. If a loader executes multiple times, it may see different data on subsequent executions.
```